feat(node): Add more detailed OTEL docs (#10872)
getsentry/sentry-docs: docs/platforms/javascript/common/opentelemetry/sampler.mdx

---

> title: Using a Custom OpenTelemetry Sampler

Reviewer: ```suggestion
title: Custom OpenTelemetry Sampler
```
It would be great if rather than making this a standalone page we could add it to either the "using a custom OTEL setup" or "using custom OTEL instrumentation" page. Would love to hear your thoughts!

> description: "How to use a custom OpenTelemetry Sampler with Sentry."

Reviewer: ```suggestion
description: "Learn how to use a custom OpenTelemetry Sampler with Sentry."
```

supported:
  - javascript.nextjs
  - javascript.node
  - javascript.aws-lambda
  - javascript.azure-functions
  - javascript.connect
  - javascript.express
  - javascript.fastify
  - javascript.gcp-functions
  - javascript.hapi
  - javascript.koa
  - javascript.nestjs
  - javascript.sveltekit
  - javascript.astro
  - javascript.remix
notSupported:
  - javascript
---

> We recommend using `SentrySampler` as this will ensure the correct subset of traces is sent to Sentry depending on your `tracesSampleRate`, as well as that all other Sentry features like trace propagation work as expected.
> If you however need to use your own sampler then make sure to wrap your `SamplingResult` with our `wrapSamplingDecision` method:

Reviewer: ```suggestion
While you can use your own sampler, we recommend that you use the `SentrySampler`. This will ensure that the correct subset of traces will be sent to Sentry, based on your `tracesSampleRate`. It will also ensure that all other Sentry features like trace propagation work as expected. If you do need to use your own sampler, make sure to wrap your `SamplingResult` with our `wrapSamplingDecision` method like in the example below:
```
